src/Util/StringUtility.php

```diff
@@ -12,4 +12,86 @@ public static function toLowerCase($string)
         return \Tools::strtolower($string);
     }
 
+    public static function trimRemarkMessage($message) {
```

Comment:
as you moved to string utility, name could be just removeSpecialCharacters or something